widget/ui/src/components/Button/Button.styles.tsx

```diff
@@ -15,6 +15,12 @@ export const ButtonBase = styled('button', {
 
   variants: {
     size: {
+      micro: {
```

Comment:
you can add x as prefix which mean you can use xxsmall instead.

Reply:
fixed.